### PR TITLE
Handle two different revisions in create_release.py (one for asserts)

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: scripts/create_release.py [hunk 1]
```diff
@@ -50,7 +53,12 @@ def main(args):
     f.write(json.dumps(release_info, indent=2))
     f.write('\n')
 
-  subprocess.check_call(os.path.join(script_dir, 'update_bazel_workspace.sh'), cwd=root_dir)
```
Comment (dschuff) [inline scripts/create_release.py:53]: this wasn't a list before? How did it ever work?

File: scripts/create_release.py [hunk 2]
```diff
@@ -50,7 +53,12 @@ def main(args):
...
+  subprocess.check_call([os.path.join(script_dir, 'update_bazel_workspace.sh')], cwd=root_dir)
+
+  branch_name = 'test_version_%s' % '_'.join(str(part) for part in new_version)
```
Comment (dschuff) [inline scripts/create_release.py:58]: do you want to take the 'test_' prefix back out again?

**Review by dschuff — APPROVED** [review]

File: scripts/create_release.py (hunk 1 above)
Comment (sbc100) [reply scripts/create_release.py:53]: Seems like subprocess.check_call can also accept a string .. I didn't know that.   Seems like it treats it like a single element list.. so you can only call programs without any args.

File: scripts/create_release.py (hunk 2 above)
Comment (sbc100) [reply scripts/create_release.py:58]: oops.. thanks.

**Pull request merged**